util/http.go

```
@@ -13,3 +14,30 @@ func CopyReq(req *http.Request) *http.Request {
 	r.Body = ioutil.NopCloser(bytes.NewBuffer(buf))
 	return &r
 }
+
+func ResponseBody(obj interface{}) []byte {
+	respBody, err := json.Marshal(obj)
+	if err != nil {
+		return []byte(`{\"errors\":[\"Failed to parse response body\"]}`)
+	}
+	return respBody
+}
+
+func ResponseOKWithBody(rw http.ResponseWriter, obj interface{}) {
+	rw.Header().Set("Content-type", "application/json")
+	rw.WriteHeader(http.StatusOK)
+	_, _ = rw.Write(ResponseBody(obj))
```

Don't ignore the error code.

---

@yasker  updated.